src/internal/m365/service/groups/backup.go

```diff
@@ -134,15 +131,26 @@ func ProduceBackupCollections(
 
 				dbcs = append(dbcs, cs...)
 			}
-
 		case path.ChannelMessagesCategory:
+			var (
+				cs                   []data.BackupCollection
+				canUsePreviousBackup bool
+				err                  error
```

Comment:
This is done so that we can capture `cs` in the closure passed to `SubMessageWithCompletionAndTip` [here](https://github.com/alcionai/corso/blob/3dec2fbd56d6901bdcfe268efd7c2a510da34098/src/internal/m365/service/groups/backup.go#L145).